louet_website/controllers/controller.py

```diff
@@ -0,0 +1,17 @@
+from odoo import http
```

Comment:
Not a big deal, but I recommend changing the python file to something like "website_sale.py"
just "controller.py" is a bit too vague.